src/cache_client.rs

```diff
@@ -556,6 +557,80 @@ impl CacheClient {
         request.send(self).await
     }
 
+    /// Fetch the elements in the given set.
+    ///
+    /// # Arguments
+    ///
+    /// * `cache_name` - The name of the cache containing the set.
+    /// * `set_name` - The name of the set to fetch.
+    ///
+    /// # Examples
+    /// Assumes that a CacheClient named `cache_client` has been created and is available.
+    /// ```
+    /// # fn main() -> anyhow::Result<()> {
+    /// # use momento::MomentoResult;
+    /// # use momento_test_util::create_doctest_cache_client;
+    /// # tokio_test::block_on(async {
+    /// use momento::cache::SetFetch;
+    /// use std::convert::TryInto;
+    /// # let (cache_client, cache_name) = create_doctest_cache_client();
+    /// let set_name = "set";
+    ///
+    /// # let add_elements_response = cache_client.set_add_elements(&cache_name, set_name, vec!["value1", "value2"]).await?;
+    ///
+    /// let fetched_elements: Vec<String> = cache_client.set_fetch(cache_name, set_name).await?.try_into().expect("Expected a set!");
+    /// # Ok(())
+    /// # })
+    /// # }
+    /// ```
+    /// You can also use the [send_request](CacheClient::send_request) method to fetch elements using a [SetFetchRequest].
+    pub async fn set_fetch(
+        &self,
+        cache_name: impl Into<String>,
+        set_name: impl IntoBytes,
+    ) -> MomentoResult<SetFetch> {
+        let request = SetFetchRequest::new(cache_name, set_name);
+        request.send(self).await
+    }
+
+    /// Removes multiple elements from an existing set. If the set is emptied as a result, the set is deleted.
```

Comment:
I also made a note that requesting to remove elements not present is not an error, and requesting to remove elements on a set that doesn't exist is not an error either.